packages/paste-core/primitives/combobox/stories/mutliselect.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import difference from 'lodash/difference';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+import {Box} from '@twilio-paste/box';
+import {Input} from '@twilio-paste/input';
+import {Label} from '@twilio-paste/label';
+import {FormPill, FormPillGroup, useFormPillState} from '@twilio-paste/form-pill-group';
+import {ComboboxListbox, ComboboxListboxGroup, ComboboxListboxOption} from '@twilio-paste/combobox';
+
+import type {StoryFn} from '@storybook/react/dist/ts3.9/client/preview/types-6-0';
```

Comment:
I suspect what you actually want is 

`import { ComponentStory, ComponentMeta } from '@storybook/react';`

as seen in the typescript examples https://storybook.js.org/docs/react/writing-stories/introduction

Hopefully that saves you from doing anything to the tsconfig files

Reply:
Huh, interesting. The other day when I added this I _tried_ to import from that package but it didn't work for some reason. 🤔 

Will update.